code-challenges.test.js

```diff
@@ -22,31 +30,79 @@ const hitchhikersCharacters = [
 // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 
 // b) Create the function that makes the test pass.
-
+  const capitalized = (input) => {
+    let firstName = input.split(' ')[0]
+    let lastName = input.split(' ')[1]
+    return (firstName[0].toUpperCase() + firstName.slice(1)) + " " + (lastName[0].toUpperCase() + lastName.slice(1))
+  }
+
+  const personInfo = (input) => {
+    return input.map((item) => {
+      return `${capitalized(item.name)} is ${item.occupation}.`
```

Comment:
Good logic, though some interviewers may dislike how it's not all in a single function, might be worth keeping in mind